cms/templates/course_outline.html

```diff
@@ -199,7 +199,8 @@ <h3 class="title-3">${_("Setting release dates and grading policies")}</h3>
                 <p>${_("Select the Configure icon for a section or subsection to set its release date. When you configure a subsection, you can also set the grading policy and due date.")}</p>
                 <h3 class="title-3">${_("Changing the content students see")}</h3>
                 <p>${_("To publish draft content, select the Publish icon for a section, subsection, or unit.")}</p>
-                <p>${Text(_("To hide content from students, select the Configure icon for a section, subsection, or unit, then select {em_start}Hide from students{em_end}.")).format(em_start=HTML("<strong>"), em_end=HTML("</strong>"))}</p>
+                <p>${Text(_("To make a section, subsection, or unit unavailable to students (including grade calculations), select the Configure icon for a section, subsection, or unit, then select {em_start}Hide from students{em_end}.")).format(em_start=HTML("<strong>"), em_end=HTML("</strong>"))}</p>
```

Comment:
@catong @sstack22 @chris-mike How does this look for multiple doc link options?

<img width="340" alt="screen shot 2016-07-20 at 12 20 05 pm" src="https://cloud.githubusercontent.com/assets/7373924/16994216/6ee8fd64-4e74-11e6-8400-d9d76b161a9b.png">